typescript/nomad-deploy/scripts/moonBaseAlpha/core.ts

```diff
@@ -0,0 +1,23 @@
+import { deployNChains } from '../../src/core';
```

Comment:
this file should be removed, i believe. we want scripts to have folders `local`, `development`, `staging`, `production`

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/ff75273178b6c024dbce8b40399362d8e5c5d680